cmd/executor/cmd/root.go

```diff
@@ -182,6 +187,7 @@ func addKanikoOptionsFlags() {
 	RootCmd.PersistentFlags().BoolVarP(&opts.SkipTLSVerifyPull, "skip-tls-verify-pull", "", false, "Pull from insecure registry ignoring TLS verify")
 	RootCmd.PersistentFlags().IntVar(&opts.PushRetry, "push-retry", 0, "Number of retries for the push operation")
 	RootCmd.PersistentFlags().IntVar(&opts.ImageFSExtractRetry, "image-fs-extract-retry", 0, "Number of retries for image FS extraction")
+	RootCmd.PersistentFlags().StringVarP(&opts.KanikoDir, "kaniko-dir", "k", "/kaniko", "Path to the kaniko directory")
```

Comment:
I don't think we need the shorthand flag. This is hopefully a pretty rare use case, and saving `-k` for something later will probably be useful.

Reply:
That makes sense, my thinking was that it's quite an impactful (maybe not?) change, so having a shorthand for it is quite nice. I agree though that there may well be something else that comes later which is best reserved for `-k`.

Shall swap this out!